[2025-06-14T17:24:37.310+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-06-14T17:24:37.342+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: usdpkr_forecasting_modular.train_stacked manual__2025-06-14T17:06:59.489541+00:00 [queued]>
[2025-06-14T17:24:37.356+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: usdpkr_forecasting_modular.train_stacked manual__2025-06-14T17:06:59.489541+00:00 [queued]>
[2025-06-14T17:24:37.357+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 3
[2025-06-14T17:24:37.374+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): train_stacked> on 2025-06-14 17:06:59.489541+00:00
[2025-06-14T17:24:37.381+0000] {standard_task_runner.py:63} INFO - Started process 1753 to run task
[2025-06-14T17:24:37.384+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'usdpkr_forecasting_modular', 'train_stacked', 'manual__2025-06-14T17:06:59.489541+00:00', '--job-id', '451', '--raw', '--subdir', 'DAGS_FOLDER/usdpkr_forecasting_dag.py', '--cfg-path', '/tmp/tmptl_4xfnx']
[2025-06-14T17:24:37.386+0000] {standard_task_runner.py:91} INFO - Job 451: Subtask train_stacked
[2025-06-14T17:24:37.489+0000] {task_command.py:426} INFO - Running <TaskInstance: usdpkr_forecasting_modular.train_stacked manual__2025-06-14T17:06:59.489541+00:00 [running]> on host 1ada37a5d2dc
[2025-06-14T17:24:37.647+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='usdpkr_forecasting_modular' AIRFLOW_CTX_TASK_ID='train_stacked' AIRFLOW_CTX_EXECUTION_DATE='2025-06-14T17:06:59.489541+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-14T17:06:59.489541+00:00'
[2025-06-14T17:24:37.649+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-06-14T17:24:37.686+0000] {model_steps.py:950} ERROR - Error in train_stacked_model: Empty data loaded. X_train shape: (0, 45), y_train shape: (0, 1)
[2025-06-14T17:24:37.687+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-06-14T17:24:37.688+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/inflation_modules/model_steps.py", line 921, in train_stacked_model
    raise ValueError(f"Empty data loaded. X_train shape: {X_train.shape}, y_train shape: {y_train.shape}")
ValueError: Empty data loaded. X_train shape: (0, 45), y_train shape: (0, 1)
[2025-06-14T17:24:37.705+0000] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=usdpkr_forecasting_modular, task_id=train_stacked, run_id=manual__2025-06-14T17:06:59.489541+00:00, execution_date=20250614T170659, start_date=20250614T172437, end_date=20250614T172437
[2025-06-14T17:24:37.719+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 451 for task train_stacked (Empty data loaded. X_train shape: (0, 45), y_train shape: (0, 1); 1753)
[2025-06-14T17:24:37.762+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-06-14T17:24:37.773+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
